.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,13 @@
+Thank you for contributing to The Foreman, please read the following guide
+available at https://www.theforeman.org/contribute.html - in short:
+
+* Create an issue at http://projects.theforeman.org/projects/foreman/issues
+* Reference the issue via "Fixes #1234" or "Refs" in the commit message
+* Run Rubocop prior sending your PR, check Coding Standards in the guide
+* Extract all strings for l18n, see Translation section in the guide
+* Prepend `[WIP]` or `WIP` to prevent lengthy tests for work in progress
```

Comment:
I'd remove this

Reply:
But this is still a pain, I propose to only remove the WIP line after our CI is fast enough.